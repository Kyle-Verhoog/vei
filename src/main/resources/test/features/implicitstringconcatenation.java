public class Implicitstringconcatenation {
  public Implicitstringconcatenation() {}
  public String m(int x) {
    return "foo" + x + true + null;
  }
}