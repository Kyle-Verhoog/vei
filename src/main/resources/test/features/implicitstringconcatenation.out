PUBLIC ┃ public
CLASS ┃ class
IDENTIFIER ┃ A
{ ┃ {
PUBLIC ┃ public
IDENTIFIER ┃ A
( ┃ (
) ┃ )
{ ┃ {
} ┃ }
PUBLIC ┃ public
IDENTIFIER ┃ String
IDENTIFIER ┃ m
( ┃ (
INT ┃ int
IDENTIFIER ┃ x
) ┃ )
{ ┃ {
RETURN ┃ return
STRING_LITERAL ┃ "foo"
+ ┃ +
IDENTIFIER ┃ x
+ ┃ +
BOOLEAN_LITERAL ┃ true
+ ┃ +
NULL_LITERAL ┃ null
; ┃ ;
} ┃ }
} ┃ }